the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  SettingsTableViewController.swift
+//  The Blue Alliance
+//
+//  Created by Anas Merbouh on 18-05-20.
+//  Copyright © 2018 The Blue Alliance. All rights reserved.
+//
+
+import UIKit
+
+class SettingsTableViewController: UITableViewController {
+    
+    // MARK: - View's lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "BaseCell")
+        tableView.register(UINib(nibName: "SwitchTableViewCell", bundle: nil), forCellReuseIdentifier: "SwitchCell")
+    }
+    
+    // MARK: - Interface Methods
+    
+    @IBAction func doneTapped(_ sender: UIBarButtonItem) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    // MARK: - Table view configuration
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 4
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0:
+            return "Account"
```

Comment:
This first and second section (the Account and the Notifications) can both go. We shouldn't be handling these at a Settings level. Enabling myTBA and enabling push notifications is all done based on authentication of myTBA.

Reply:
If we remove these sections, we probably don't need the switch code/xib as well